overlord/ifacestate/handlers.go

```diff
@@ -1650,7 +1653,7 @@ func (m *InterfaceManager) doHotplugSeqWait(task *state.Task, _ *tomb.Tomb) erro
 		// conflict with retry if there another change affecting same device and has lower sequence number
 		if hotplugKey == otherKey && otherSeq < seq {
 			task.Logf("Waiting processing of earlier hotplug event change %q affecting device with hotplug key %q", otherChg.Kind(), hotplugKey)
-			return &state.Retry{}
+			return &state.Retry{After: hotplugRetryTimeout}
```

Comment:
Can you explain this a little? What is the backstory for the retry time?

Reply:
With no timeout specified and if there are no other tasks left for current iteration of Ensure - AFAIU - taskrunner may reconsider this task on next ensure interval (which AFAIR is 5 minutes). I observed this in the attached spread test. 
We do set a specific timeouts on other types of conflicts and we should to the same here, with relatively low value.

Reply:
It would be good to have an idea how long the hotplug changes tend to take. A different solution would be to end all hotplug changes with a task that does EnsureBefore(0)

Reply:
I'm not sure this can be measured, these changes may involve running connect/disconnect hooks, their performance is up to snaps?
The idea of `EnsureBefore(0)` is fine, but I'm concerned this would need to appear in at least 2 task kinds already, if not more - easy to forget about it if code evolves, unless we add an artificial task that does exactly this, which seems a bit over the top.

Reply:
yes, my idea would be to add an artificial task like hotplug-seq-wait, it might be overkill.  Maybe just reduce the timeout time to 300ms,  and add a TODO here about the dedicated end task idea.

Reply:
Done.